src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.model.behavior.Areable;
+import core.basesyntax.model.behavior.Drawable;
+import core.basesyntax.model.state.ColorSupplier;
+
+public abstract class Figure implements Areable, Drawable {
+    private String identification;
+    private String color;
+
+    public Figure() {
+        color = ColorSupplier.generateColor();
```

Comment:
Do not initialize variables with random values in the constructor. Pass them from the outside